packages/constants/src/mcf.ts

```diff
@@ -49,6 +49,9 @@ const mapMsfToMcfEmploymentTypes = () => {
       if (employmentType.ilpDescription === 'Contract Basis') {
         return {id: parseInt(employmentType.ilpId, 10), employmentType: 'Contract'};
       }
+      if (employmentType.ilpDescription === 'Internship') {
```

Comment:
Please add story link in comment, to explain change and stating this as special case

Reply:
okie

Reply:
```
// NOTE: https://www.pivotaltracker.com/n/projects/2434214/stories/172194713
```

Reply:
Even butter,

```
// NOTE: part of the support for the recently announced SGUnitedTraineeship, https://www.pivotaltracker.com/n/projects/2434214/stories/172194713
```

Reply:
Nit.